src/applications/vaos/tests/project-cheetah/components/ContactInfoPage.unit.spec.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import { expect } from 'chai';
+import userEvent from '@testing-library/user-event';
+
+import ContactInfoPage from '../../../project-cheetah/components/ContactInfoPage';
+import { createTestStore, renderWithStoreAndRouter } from '../../mocks/setup';
+import { FETCH_STATUS } from '../../../utils/constants';
+import { cleanup } from 'axe-core';
+
+describe('VAOS <ContactInfoPage>', () => {
+  const initialState = {
+    projectCheetah: {
+      newBooking: {
+        data: {},
+        pages: [],
+        previousPages: [],
+      },
+    },
+  };
+
+  it('should submit with valid data', async () => {
+    const store = createTestStore(initialState);
+
+    let screen = renderWithStoreAndRouter(
+      <Route component={ContactInfoPage} />,
+      {
+        store,
+      },
+    );
+
+    let input = await screen.findByLabelText(/^Your phone number/);
+    userEvent.type(input, '5555555555');
+
+    input = screen.getByLabelText(/^Your email address/);
+    userEvent.type(input, 'joe.blow@gmail.com');
+
+    // it should display page heading
+    expect(screen.getByText('Confirm your contact information')).to.be.ok;
+    const button = await screen.findByText(/^Continue/);
+
+    userEvent.click(button);
+    expect(screen.history.push.called).to.be.true;
+
+    // Expect the previously entered form data is still there if you unmount and remount the page with the same store,
+    await cleanup();
+    screen = renderWithStoreAndRouter(<Route component={ContactInfoPage} />, {
+      store,
+    });
+
+    input = await screen.findByLabelText(/^Your phone number/);
+    expect(input.value).to.equal('5555555555');
+
+    input = screen.getByLabelText(/^Your email address/);
+    expect(input.value).to.equal('joe.blow@gmail.com');
+  });
+
+  it('should not submit empty form', async () => {
```

Comment:
Often we'll also have a test that makes changes to date, then remounts the page with the same redux store, to make sure data is saved between page refreshes. I think that'd be useful here, so we can make sure the prefilling logic doesn't overwrite user entered info.